kafka/admin/client.py

```diff
@@ -205,9 +208,11 @@ def __init__(self, **configs):
         reporters = [reporter() for reporter in self.config['metric_reporters']]
         self._metrics = Metrics(metric_config, reporters)
 
-        self._client = KafkaClient(metrics=self._metrics,
-                                   metric_group_prefix='admin',
-                                   **self.config)
+        self._client = get_client_factory(self.config)(
```

Comment:
I don't love the `get_client_factory` indirection. Could we skip that and just use the config lookup directly?

Reply:
removed